paddle2onnx/op_mapper/nn.py

```diff
@@ -23,25 +23,26 @@
 from paddle2onnx import utils
 
 
-@op_mapper(['conv2d', 'depthwise_conv2d'])
+@op_mapper(['conv2d', 'depthwise_conv2d', 'conv3d'])
 class Conv():
     support_opset_version_range = (1, 12)
 
     @classmethod
     def opset_1(cls, graph, node, **kw):
         kernel_shape = node.input_shape('Filter', 0)
         dilations = node.attr('dilations')
-        kernel_shape = kernel_shape[-2:]
+        kernel_shape = kernel_shape[2:]
         strides = node.attr('strides')
         group = node.attr('groups')
         pads = node.attr('paddings')
-        assert node.attrs['data_format'] == 'NCHW', "The conv data format should be 'NCHW', but received data format " \
-                                                    "is %s." % node.attrs['data_format']
+
```

Comment:
assert is required,  only support NCHW/NCDHW now

Reply:
done, add assert